# Taken from Jasan's control_utils package as is to remove minor dependency
Header header
float64[] vector